packages/shared/tailwind.config.js

```diff
@@ -1,12 +1,71 @@
 module.exports = {
-  content: [
-    './components/**/*.{js,ts,jsx,tsx}',
+  content: ['./components/**/*.{js,ts,jsx,tsx}'],
+  safelist: [
```

Comment:
what is this for? do we need 'text-X', 'bg-X" as well?

Reply:
Might be something we could do with https://tailwindcss.com/docs/content-configuration#using-regular-expressions
but in general the way tailwind works kind of leads to some jank on the safelist if you have dynamic classes anywhere